reports/Pennsylvania.md

```diff
@@ -66,16 +66,32 @@ Three protestors kneeling on the ground with their hands on their heads/covering
 
 * https://twitter.com/d0wnrrrrr/status/1267691766188310528
 
+### Police kneel on man who says "I can't breathe", officer yells profanities back | believed to be June 3rd
+
+At least 6 officers surround a handcuffed man who says "I can't breathe". Officer replies "Shut up asshole. Are you fucking stupid? Shut up, that shit don't work here."
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268200800649707526
+
+### Police shove protestors and strike woman across the face with a baton
+
+Police shove a group of protestors. One officer in a white shirt runs and shoves someone and is then approached by a woman who he strikes across the face with a baton before jumping on her and cuffing her.
+
+**Links**
+* https://twitter.com/greg_doucette/status/1268766216161763328
```

Comment:
I don't get why this isn't causing a merge conflict with the current contents of `Pennsylvania.md` where this incident is already reported in the same position in the file. Keen to get this PR merged though for the other contributions.

Reply:
Seems like it should conflict with this: https://github.com/2020PB/police-brutality/blob/e49fbf8706ab14e08e01682ed5d4c09d5abf2fee/reports/Pennsylvania.md#philadelphia-police-disperse-crowd-with-batons--june-1st

Reply:
@rote-culpability I'm gonna merge this PR and then sort out any duplication afterwards :+1: